reference/masterfiles-policy-framework/cfe_internal-enterprise-federation-federation.markdown

```diff
@@ -5,4 +5,37 @@ published: true
 tags: [reference, MPF, cfe_internal, federated reporting]
 ---
 
+This policy file handles Federated Reporting setup and ongoing operations.
+
+There are several augments which allow the tuning of this policy:
+
+- `cfengine_mp_fr_dependencies_auto_install` class
+
+  If defined and selinux is enabled then this policy will install the `semanage` utility for use in managing directories and files for the import process.
```

Comment:
I think this should say that the policy will install all non-shipped dependencies needed for FR. And that right now, it's just `semanage`.